activestorage/db/update_migrate/20191206030411_create_active_storage_variant_records.rb

```diff
@@ -1,11 +1,17 @@
 class CreateActiveStorageVariantRecords < ActiveRecord::Migration[6.0]
   def change
-    create_table :active_storage_variant_records do |t|
-      t.belongs_to :blob, null: false, index: false
+    create_table :active_storage_variant_records, id: primary_key_type do |t|
```

Comment:
https://github.com/rails/rails/blob/main/activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb#L296 -> the default argument is `:primary_key`, I think that's what should be provided here as a fallback. Currently your fallback value would be `:bigint`.